docs/marketplace/add_to_marketplace/add_item_by_type/add_plugin.md

```diff
@@ -309,7 +308,16 @@ You just created your plugin, which is now available on your `Company Marketplac
 
 You can update a Custom Resource Marketplace item by using the same `miactl marketplace apply` explained before, by including an updated file (either in `json` or `yaml` format).
 
-While non-versioned items can be modified in place without any limitation (except for `itemId` and `tenantId` that identifies the resource to update), versioned Marketplace Custom Resource have some fields that cannot be modified. For more information, refer to the [dedicated section on the "Create your Company Marketplace" page](/marketplace/add_to_marketplace/create_your_company_marketplace.md#editing-a-versioned-resource).
+While non-versioned items can be modified in place without any limitation (except for `itemId` and `tenantId` that identifies the resource to update), versioned Marketplace Custom Resource have some fields that cannot be modified.
+
+If you want to do so, you must create a new version of the resource: for more information, refer to the [dedicated section on the _Create your Company Marketplace_ page](/marketplace/add_to_marketplace/create_your_company_marketplace.md#how-to-create-a-new-version-of-your-resource).
 
 Also remember that versioned elements does not allow the update of the `dockerImage` field, since the value is automatically tied with the released version: if you need to change the Docker Image, you should create a new version of the resource.
+
+:::info
+Versions are not mandatory but highly suggested to avoid overwriting existing resources.
```

Comment:
```suggestion
Versions are not mandatory but highly suggested to avoid overwriting existing resources.
```

Reply:
I'm not catching the difference here. Missing something?

Reply:
fixed commend, i'd use recommended rather than suggested